.circleci/config.yml

```diff
@@ -24,34 +24,31 @@ commands:
         only_for_branches: master
         webhook: $SLACK_GREENKEEPER_WEBHOOK
   notify-rc-tag:
-    description: 'Notifies #eng-magento1 when a release candidate is tagged.'
+    description: 'Notify #eng-magento1 after a release candidate is tagged.'
     parameters:
       tag:
         type: string
         default: ''
     steps:
     - slack/notify:
-        channel: "#eng-magento1"
-        color: '#33ff00'
-        mentions: '@oleksii,@Pavel,'
+        color: '#58a359'
+        mentions: 'QAEng'
         message: ":white_check_mark: A new RC << parameters.tag >> has been tagged for Magento 1!"
-        webhook: $SLACK_GREENKEEPER_WEBHOOK
+        webhook: $SLACK_MAGENTO1_WEBHOOK
+        include_job_number_field: false
+        include_project_field: false
+        include_visit_job_action: false
 jobs:
-  tag-rc-candidate:
-    description: Tag master as rc every 3 weeks
+  tag-release-candidate:
     docker:
     - image: cimg/base:2020.01
     steps:
+    - checkout
     - run:
-        name: Validate eligibility to tag rc
-        command: .circleci/scripts/rc-tag.sh
-    - when:
-        condition: $TAGTHISWEEK
-        steps:
-        - swissknife/publish_github_release:
-            tag: $NEWTAGNAME
-        - notify-rc-tag:
-            tag: $NEWTAGNAME
+        name: Tag master as release candidate every 3 weeks
+        command: .circleci/scripts/tag-rc-if-necessary.sh
```

Comment:
the only thought i have is do we want to go purely on schedule, or do we want to notify that  we are about to cut a tag, wait for approval and then cut a tag. That is easily possible, not sure exactly what eorkflow we are looking for.

Reply:
I think completely automatic is fine. Our release schedule is fixed and we only need approval for the final release, not the initial tag.